test/data.jl

```diff
@@ -1,4 +1,10 @@
 @testset "test data handling and parsing" begin
+    @testset "gaslib40 parsing from gaslib zip" begin
+        data = GasModels.parse_file("../test/data/gaslib/GasLib-40.zip")
+        @test length(data["delivery"]) == 29
+        @test length(data["receipt"]) == 3

```

Comment:
somewhere can you also make a simple test of a model with the prv for the constraint?  I want to make sure that code is covered.

Reply:
Okay, I've added more components to the GasLib tests, which should cover all except the (empty) MIP versions of these constraints.

Reply:
Ok, this looks good